src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -303,11 +303,15 @@ public async Task<DicomWebResponse> DeleteAsync(Uri requestUri, CancellationToke
             return new DicomWebResponse(response);
         }
 
-        public async Task<DicomWebAsyncEnumerableResponse<DicomDataset>> QueryAsync(string requestUri, CancellationToken cancellationToken)
+        [Obsolete("Please use QueryAsync(Uri, CancellationToken) instead.")]
```

Comment:
We can change this method to use `Uri` in a backwards compatible manner. But we can also just delete the `string` overload.

Reply:
I think we can leave it for now. I do wonder if we should make the documentation that says it needs to be relative.